frontend/packages/dev-console/src/components/helm/HelmReleaseDetailsPage.tsx

```diff
@@ -0,0 +1,80 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { match as RMatch } from 'react-router';
+import {
+  navFactory,
+  ResourceSummary,
+  SectionHeading,
+  LoadingBox,
+  StatusBox,
+  FirehoseResult,
+} from '@console/internal/components/utils';
+import { SecretModel } from '@console/internal/models';
+import { ErrorPage404 } from '@console/internal/components/error';
+import { DetailsPage } from '@console/internal/components/factory';
+import { K8sResourceKindReference, K8sResourceKind } from '@console/internal/module/k8s';
+
+const SecretReference: K8sResourceKindReference = 'Secret';
+const HelmReleaseReference = 'HelmRelease';
+
+export interface HelmReleaseDetailsPageProps {
+  match: RMatch<{
+    ns?: string;
+  }>;
+  secret?: FirehoseResult;
+}
+
+export interface HelmReleaseDetailsProps {
+  obj: K8sResourceKind;
+}
+
+export const HelmReleaseDetails: React.FC<HelmReleaseDetailsProps> = ({ obj: resourceDetails }) => {
+  const helmRelease = resourceDetails;
+  helmRelease.metadata.name = _.get(resourceDetails, 'metadata.labels.name');
```

Comment:
Why are you changing the name of data. Not to mention this is in a render function.

Data should be considered immutable because it'll also be stored in redux.

Reply:
removed this and sending `customPathName` prop to ResourceSummary